distro/depends/makepkg

```diff
@@ -1,9 +1 @@
-curl
-bsdtar
-bzip2
-fakeroot
-gnupg
-gettext
-openssl
-ncurses-bin
-binutils
+
```

Comment:
Seems these packages are removed here and added back in later commit. How about rebase the code so the change is minimum?

Reply:
what is the purpose of removing these depends?

Reply:
ping?

Reply:
> ping?

kulkarniankita8@gmail.com